rasa/server.py

```diff
@@ -316,6 +316,17 @@ def _create_emulator(mode: Optional[Text]) -> NoEmulator:
     return loaded_agent
 
 
+def configure_cors(app: Sanic) -> None:
```

Comment:
you need to pass in the `cors_origins` part

Reply:
haha sorry wasn't quite done yet 😋le whoops